docs/CONTRIBUTE.md

```diff
@@ -0,0 +1,55 @@
+This repository is the deployed version of the main repository of Software Discovery Tool as a submodule and the data repository as a sub submodule.
+Any contribution to SDT should be done in the main repository.
+
+# How To Deploy
+The whole SDT deployment is maintained by:
+- [cicd.yml]() workflow
+- [config_build.py]() to create new sections in the distro config file
+- [package_build.py]() to update existing distro files to latest versions
```

Comment:
For both confg_build.py and package_build.py we need to use the full link to software-discovery-tool/config_build.py since that's where it will actually end up.